spec/acceptance/active_record_integration_spec.rb

```diff
@@ -0,0 +1,80 @@
+require 'acceptance_spec_helper'
+
+describe 'shoulda-matchers integrates with active record' do
+  before do
+    create_active_record_project
+
+    write_file 'Rakefile', <<-FILE
+      require 'active_record'
+      require 'sqlite3'
+
+      namespace :db do
+        desc 'Create the database'
+        task :create do
+          File.unlink 'test.sqlite3' if File.exist?('test.sqlite3')
+          db = SQLite3::Database.new('test.sqlite3')
+          db.execute("CREATE TABLE users (id integer)")
+          db.execute("CREATE TABLE profiles (id integer, user_id integer)")
+        end
+      end
+    FILE
+
+    run_rake_tasks!('db:create')
+
+    write_file 'lib/user.rb', <<-FILE
+      require 'active_record'
+
+      class User < ActiveRecord::Base
+      end
+    FILE
+
+    write_file 'lib/profile.rb', <<-FILE
+      require 'active_record'
+      require 'user'
+
+      class Profile < ActiveRecord::Base
+        belongs_to :user
+        validates_presence_of :user
+      end
+    FILE
+
+    write_file 'spec/profile_spec.rb', <<-FILE
+      require 'spec_helper'
+      require 'profile'
+
+      describe Profile do
+        before do
+          ActiveRecord::Base.establish_connection(adapter: 'sqlite3', database: 'test.sqlite3')
+        end
+
+        it { should validate_presence_of(:user) }
+      end
+    FILE
+
+    updating_bundle do
+      add_rspec_to_project
+      add_shoulda_matchers_to_project(
+        manually: true,
+        with_configuration: false,
+      )
+      write_file 'spec/spec_helper.rb', <<-FILE
+        require 'shoulda/matchers'
+
+        RSpec.configure do |config|
```

Comment:
I agree. 

I tried to make it work with the second option, but the problem is that the shoulda matches configuration is being added to the `rails_helper.rb` instead of `spec_helper.rb`.

I think I found the problem, but I'm not sure how to fix it. I'll keep trying.

FWIW, the problem is that is finding `rspec-rails` in the Gemfile because it's checking on the wrong Gemfile - gemfiles/rails_6_0.

https://github.com/thoughtbot/shoulda-matchers/blob/bf850fa0439d1a5a9e135e0f5434ed6e04fc2f46/spec/support/acceptance/adds_shoulda_matchers_to_project.rb#L101-L108

Values obtained in the file above using pry-byebug:

```
[1] pry(#<AcceptanceTests::AddsShouldaMatchersToProject>)> bundle
=> #<Tests::Bundle:0x0000555b8b6bf800 @already_updating=false, @fs=#<Tests::Filesystem:0x0000555b8b6bf7d8>>
[2] pry(#<AcceptanceTests::AddsShouldaMatchersToProject>)> bundle.includes?('rspec-rails')
=> true
[3] pry(#<AcceptanceTests::AddsShouldaMatchersToProject>)> ENV['BUNDLE_GEMFILE']
=> "<path-to>/shoulda-matchers/gemfiles/rails_6_0.gemfile"
[4] pry(#<AcceptanceTests::AddsShouldaMatchersToProject>)> 
```

Reply:
In a "normal" app, yes, it would make sense for BUNDLE_GEMFILE to be the same as the application's Gemfile. For the tests we do a sort of hack where we force it to be the Appraisal gemfile instead. The reason we do that is because we want the Appraisal to dictate the list of gems that are loaded, as that is something we can control. As far as how BUNDLE_GEMFILE gets set, that comes from Appraisal. The 6.0 gemfile is the default because that's the latest one, but you  You can override the Gemfile being used by using the `appraisal` command:

``` ruby
bundle exec appraisal rails_5_2 rspec ...
```

Reply:
I get it. Thanks!

I am thinking of using the first approach you suggested. I couldn't get it to work with the second.

What do you think?

Reply:
Oh that's fine to use the first option! I didn't realize it only added to `rails_helper`.

Reply:
Thanks! Merging!